.eslintrc

```diff
@@ -43,14 +43,14 @@ rules:
   no-shadow: 2
   no-multi-spaces: 2
 
-  # WARNINGS
-  # This is the only one that's hard to track since we don't lint just changes.
-  max-len: [1, 80]
-
   # WISHLIST. One day...
   # We'll need a custom version of this that does a subset of the whole rule.
   # Otherwise this is just too noisy.
   # valid-jsdoc: 1
+  # Ideally, we could just warn when *new* lines are added that exceed 80 chars
+  # while not warning about existing ones (often URLs, etc. which are
```

Comment:
That would be nice functionality to have, but I don't think it would help us much: In the current state, we've got *hundreds* of warnings, and many of those are not containing URLs.

Ideally we should fix them, or change our style rules, but we're having trouble coming to a consensus internally, and this fix is more immediately important, as the large number of warnings makes it hard to effectively use the lint tool.